docs/setuptools.txt

```diff
@@ -2293,6 +2293,44 @@ Metadata and options are set in the config sections of the same name.
 
 * Unknown keys are ignored.
 
+setup.cfg-only projects
+=======================
+
+.. note:: New in 40.9.0 (03 Apr 2019).
+
+Latest versions of ``setuptools`` emulate a dummy ``setup.py`` module
+containing only a ``setuptools.setup()`` call.
+
+This means that you can have a Python distribution project with no
+``setup.py`` but having only ``setup.cfg`` present, given you **can rely
+on** having ``setuptools >= 40.9.0`` being present prior the build.
```

Comment:
Gramatically speaking it should be "prior **to** the build.", but it also seems somewhat unnecessary because while PEP 517 and PEP 518 are two different things, in practice anything that supports PEP 517 will also support PEP 518, so you really just need to specify a dependency on `setuptools >= 40.9.0`.

That said, even if you specify a dependency on `setuptools`, `pip` will just pull the latest version (since it's a clean build anyway), so practically speaking any PEP 517/518 frontend will likely just work by default.